### PR TITLE
Delete the implementation of DML backend and disable CI

### PR DESCRIPTION
**Pull request by mingmingtasd**

### CODE REVIEW

**Comment by mingmingtasd** [comment]
PTAL, I will rebase PR https://github.com/webmachinelearning/webnn-native/pull/287 on this. @fujunwei @bbernhar

File: src/webnn/native/BUILD.gn [hunk 1]
```diff
@@ -218,8 +218,6 @@ source_set("sources") {
       "dml/ContextDML.h",
       "dml/GraphDML.cpp",
       "dml/GraphDML.h",
-      "dml/DMLUtils.cpp",
```
Comment (fujunwei) [inline src/webnn/native/BUILD.gn:221]: Please delete all the files in the sources and then add them piece by piece.

**Review by fujunwei — APPROVED** [review]
LGTM, merge it.

**Comment by BruceDai** [comment]
@mingmingtasd Please also update `README.md` by removing `DML` information from CI status table, thanks.

**Pull request merged**